lib/req/steps.ex

```diff
@@ -736,12 +736,16 @@ defmodule Req.Steps do
   | 301, 302, 303 | Changed to GET     |
   | 307, 308      | Method not changed |
 
+  If we reach the maxium number of redirects, we will raise an exception.
+
   ## Request Options
 
     * `:location_trusted` - by default, authorization credentials are only sent
       on redirects to the same host. If `:location_trusted` is set to `true`, credentials
       will be sent to any host.
 
+    * `:max_redirects` - the maximum number of redirects, defaults to `10`. 
```

Comment:
please mention this on Req.request/1 too. After that we're ready to go, thanks!

Reply:
Shout we add `location_trusted` too?

Reply:
Yup, good catch.